### ISSUE TITLE
Lando Init fails with "cp: cannot create regular file 'filename': File exists

### ISSUE THREAD

**Issue by wernerglinka** [lando/lando#2275]
Lando version: 3.0.1
Docker version: 2.2.0.5

Installed Lando

`lando init --source pantheon`

Then it fails like this  

```
Creating network "landoinitrde01_default" with the default driver
Creating volume "landoinitrde01_data_init" with default driver
Creating volume "landoinitrde01_home_init" with default driver
Creating landoinitrde01_init_1 ... done
Killing landoinitrde01_init_1 ... done
Starting landoinitrde01_init_1 ... done
Killing landoinitrde01_init_1 ... done
Starting landoinitrde01_init_1 ... done
Detected that ssh://codeserver.dev.aaf1fc78-29d6-4d2b-bca5-61794c3300e1@codeserver.dev.aaf1fc78-29d6-4d2b-bca5-61794c3300e1.drush.in:2222/~/repository.git is a git repo
Cloning into '.'...
remote: Counting objects: 55022, done.
remote: Compressing objects: 100% (19485/19485), done.
remote: Total 55022 (delta 39884), reused 49857 (delta 35147)
Receiving objects: 100% (55022/55022), 59.03 MiB | 5.67 MiB/s, done.
Resolving deltas: 100% (39884/39884), done.
Checking connectivity... done.
Copying git clone over to /app...
cp: cannot create regular file '/app/wp-content/themes/rde01/js/jquery.hoverintent.min.js': File exists
Killing landoinitrde01_init_1 ... done

```

I also tried the pantheon localdev app and it fails the same way for the same site.

However, the site files are all downloaded and the file it is complaining about is there as well??

Any idea what's going on?

**Comment by krishnagopinath** [comment]
I'm facing the same issue as well. Some help would be greatly appreciated! @wernerglinka re: files being downloaded and still getting complaints from the CLI. Maybe there is a race condition in the script somewhere?

**Comment by npoku** [comment]
Bump. I too I'm having this problem.

**Comment by wernerglinka** [comment]
@krishnagopinath @npoku  I am using Pantheon as my hosting provider. I looked at the code directory on Pantheon and found that there are 2 files with the same name. The issue was the file `jquery.hoverintent.min.js`. The original file from npm was named `jquery.hoverIntent.min.js`. I guess something to do with the fact that there is a capital letter in the original file name. I deleted the file with the capital letter in it and that was it.
